cmd/util/client.go

```diff
@@ -90,12 +90,12 @@ func GetImpersonatedClient(userName string) (crclient.Client, error) {
 	return client, nil
 }
 
-// ParseAWSTags does exactly that
-func ParseAWSTags(tags []string) (map[string]string, error) {
+// ParseTags converts []string to map[string]string
+func ParseTags(tags []string) (map[string]string, error) {
```

Comment:
Are there any forbidden tags that could interfere with CAPI normal workflow? 

Context: I'm saying this because of the bug opened in ROSA in AWS resources which contains a tag `name=<cluster-name>` (IIRC) and that caused an issue with CAPI and CAPI-AWS trying to find an new fresh instance and being recreated in loop because of that tag.

Reply:
Im not sure,  are you suggesting we should check that no forbidden key=value pairs are being specified ?